src/bindings/python/src/pyopenvino/utils/utils.hpp

```diff
@@ -5,25 +5,34 @@
 #pragma once
 
 #include <pybind11/pybind11.h>
-#include <openvino/core/any.hpp>
-#include <openvino/core/type/element_type.hpp>
-#include <openvino/runtime/properties.hpp>
+
+#include "openvino/core/any.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "openvino/runtime/properties.hpp"
+#include "openvino/frontend/decoder.hpp"
 
 namespace py = pybind11;
 
 namespace Common {
 namespace utils {
-    py::object from_ov_any(const ov::Any &any);
+py::object from_ov_any(const ov::Any& any);
 
-    std::map<std::string, ov::Any> properties_to_any_map(const std::map<std::string, py::object>& properties);
+std::map<std::string, ov::Any> properties_to_any_map(const std::map<std::string, py::object>& properties);
```

Comment:
Please use `std::unordered_map`

Reply:
That function is out of scope of this PR, we can change it separately